### DISCUSSION TITLE
Serious lag in popover

### DISCUSSION THREAD

**Discussion by 9mm** [unovue/reka-ui#1000] [Q&A]
Im unsure of if this is a bug, however it's really quite bad, and I have checked other component libraries. Radix UI doesnt have this issue.

I also see it on the radix-vue homepage demos.

Edit this is in latest Chromium

https://github.com/radix-vue/radix-vue/assets/1386207/705b082e-f6c4-4af0-bc8b-4e4088dd5b80

https://github.com/radix-vue/radix-vue/assets/1386207/88293276-7b9f-4447-b35f-54dfe4948461

**Comment by zernonia** [comment]
Hey @9mm , what kind of "lag" your are referring ya? If you mean the popover delaying the position when scrolling, the same happened in Radix UI. Definitely something we can improve! 😁

**Reply by 9mm** [reply]
Thanks @zernonia  !!

So i think its because the other radix UI uses absolute positioning in floating UI. I dont experience the same problem on floating UI.

I tried using the always/optimized setting, but setting it to `always` actually made the stuttering worse


https://github.com/radix-vue/radix-vue/assets/1386207/d072e030-4741-4844-bff8-403edd9a8f61


https://github.com/radix-vue/radix-vue/assets/1386207/2e56c066-3150-4fca-9396-c3471a696e30

**Reply by 9mm** [reply]
Actually, let me provide a bit more context @zernonia 

If you go to Radix UI, you can see they actually use the same `fixed` positioning in their component... like this:

<img width="976" alt="image" src="https://github.com/radix-vue/radix-vue/assets/1386207/01eb37f9-08d5-427f-ba96-a31523e6675a">

BUT...... if you go to their demo page here:

https://www.radix-ui.com/primitives

Theyre actually OVERRIDING it with some style, thats probably specific to the demo page.

<img width="1304" alt="image" src="https://github.com/radix-vue/radix-vue/assets/1386207/e97414b6-bb8a-4dc3-9ceb-7bae0c0b8acd">

This is funny to me that the performance isnt good so they do some hardcoded styling.

I tried adding absolute styling to make it smooth but it breaks everything

I know this is possible, because floating-ui offers this, and all the floating UI demos are perfect

**Reply by 9mm** [reply]
This here https://github.com/radix-vue/radix-vue/blob/5da9e7b58a3f053881d1efe5b2e3eefd593e4687/packages/radix-vue/src/Popper/PopperContent.vue#L277

https://floating-ui.com/docs/usefloating#strategy

Im honestly very confused how this is like this in radix-ui as well, and how there are no github issues about it. the performance is just terrible compared to absolute position performance. You'd think they would have exposed that as an option a long time ago 🤔
